### ISSUE TITLE
0.1.25 fails to configure: ValueError: 'pyedflib/_extensions/_pyedflib.pyx' doesn't match any files

### ISSUE THREAD

**Issue by yurivict** [holgern/pyedflib#168]
```
===>  Configuring for py38-pyEDFlib-0.1.25
setup.py:132: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 263, in <module>
    ext_modules = cythonize(ext_modules, compiler_directives=cythonize_opts)
  File "/usr/local/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 966, in cythonize
    module_list, module_metadata = create_extension_list(
  File "/usr/local/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 816, in create_extension_list
    for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
  File "/usr/local/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 114, in nonempty
    raise ValueError(error_msg)
ValueError: 'pyedflib/_extensions/_pyedflib.pyx' doesn't match any files
*** Error code 1
```

FreeBSD 13

**Comment by skjerns** [comment]
1) have you tried installing the develop version?
2) We are currently not actively supporting FreeBSD, partially due to lack of testing systems

**Comment by yurivict** [comment]
The GitHub version 0.1.25 builds fine, but the PyPI version 0.1.25 fails.

There's some difference.

**Comment by nik-sm** [comment]
I'm encountering the same issue. I tried installing on Ubuntu 20.04 and on macOS 12.2.

**Comment by holgern** [comment]
Should be fixed in 0.1.26, I added a MANIFEST.in file which adds the missing pyx files into the source dist.

**Comment by yurivict** [comment]
Now it fails like this:
```
===>  Configuring for py38-pyEDFlib-0.1.26
Compiling pyedflib/_extensions/_pyedflib.pyx because it changed.
[1/1] Cythonizing pyedflib/_extensions/_pyedflib.pyx
/usr/local/lib/python3.8/site-packages/Cython/Compiler/Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: /wrkdirs/usr/ports/misc/py-pyEDFlib/work-py38/pyedflib-0.1.26/pyedflib/_extensions/_pyedflib.pyx
  tree = Parsing.p_module(s, pxd, full_module_name)
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 294, in <module>
    version=get_version_info()[0],
  File "setup.py", line 133, in get_version_info
    version = importlib.import_module('pyedflib.version', 'pyedflib/version.py')
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/wrkdirs/usr/ports/misc/py-pyEDFlib/work-py38/pyedflib-0.1.26/pyedflib/__init__.py", line 11, in <module>
    from ._extensions._pyedflib import *
ModuleNotFoundError: No module named 'pyedflib._extensions._pyedflib'
*** Error code 1
```

**Comment by holgern** [comment]
Sorry, but it should be fixed now in 0.1.28, could you test again?

**Comment by yurivict** [comment]
0.1.28 builds fine, thanks.
